modules/nf-core/pbmm2/align/main.nf

```diff
@@ -0,0 +1,45 @@
+process PBMM2_ALIGN {
+    tag "$meta.id"
+    label 'process_single'
+
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pbmm2:1.14.99--h9ee0642_0':
+        'biocontainers/pbmm2:1.14.99--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(bam)
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    pbmm2 align $fasta $bam ${prefix}.pbmm2.bam --num-threads ${task.cpus}
```

Comment:
$args needs to be in there as well, and I would remove the `.pbmm2` in the file name. If someone wants to have it, they can add it through prefix. 

```suggestion
    pbmm2 \\
        align \\
        $args \\
        $fasta \\
        $bam \\
        ${prefix}.bam \\
        --num-threads ${task.cpus}
```